presentation/backend/src/presentation-backend/PresentationManager.ts

```diff
@@ -517,9 +525,19 @@ export class PresentationManager {
    * @public
    */
   public async getContent(requestOptions: WithCancelEvent<Prioritized<Paged<ContentRequestOptions<IModelDb, Descriptor | DescriptorOverrides, KeySet, RulesetVariable>>>> & BackendDiagnosticsAttribute): Promise<Content | undefined> {
-    const content = await this._detail.getContent(requestOptions);
+    const shouldFormatValues = !requestOptions.omitFormattedValues;
+    const content = await this._detail.getContent({
+      ...(shouldFormatValues && this.props.schemaContextProvider !== undefined ? { omitFormattedValues: true } : undefined),
+      ...requestOptions,
```

Comment:
there seems to be a problem when `requestOptions.omitFormattedValues === false && this.props.schemaContextProvider !== undefined`